plugins/mattermost/client/utils/mattermost.ts

```diff
@@ -0,0 +1,77 @@
+import { userDetailsUrl as userDataUrl, userTeamsUrl } from "../../shared/url";
+import { UserData, TeamData } from "./types";
+import { Dictionary } from "./useDictionary";
+
+export class MattermostClient {
+  #serverUrl: string;
```

Comment:
```suggestion
  private serverUrl: string;
```

Prefer private and public keywords for these